### ISSUE TITLE
E: Unable to locate package libmysql-java tomcat7

### ISSUE THREAD

**Issue by lukaszertel** [MysticRyuujin/guac-install#104]
After run script on Ubuntu Server 19.10 64bit it ends with
```
Installing dependencies. This might take a few minutes...
Failed. See /tmp/guacamole_1.1.0_build.log
```
Log file shows
```
E: Unable to locate package libmysql-java
E: Unable to locate package tomcat7
```
File /etc/apt/sources.list attached (universe and multiverse enabled)
[sources.list.txt](https://github.com/MysticRyuujin/guac-install/files/4200404/sources.list.txt)

Any ideas?

**Comment by MysticRyuujin** [comment]
Does this work?
`sudo add-apt-repository universe`

**Comment by MysticRyuujin** [comment]
There's a reddit post that suggests we need to change to `libmariadb-java` for 19.10 but tomcat7 I'm not sure, can you run a: `apt-cache show tomcat7`

**Comment by lukaszertel** [comment]
Dear Chase,
I’ve checked your suggestions, results below.
I don’t insist to use Ubuntu Server 19.10 – please tell me which distro you are using (which distro working with your scrip without issues).

Cheers!
Lucas

osboxes@osboxes:~$ sudo apt-cache show tomcat7
[sudo] password for osboxes:
N: Unable to locate package tomcat7
E: No packages found

osboxes@osboxes:~$ sudo add-apt-repository universe
'universe' distribution component is already enabled for all sources.



Wysłane z aplikacji Poczta<https://go.microsoft.com/fwlink/?LinkId=550986> dla Windows 10


Od: Chase Wright <notifications@github.com>
Wysłane: Thursday, February 13, 2020 11:37:40 PM
Do: MysticRyuujin/guac-install <guac-install@noreply.github.com>
DW: lukaszertel <lukasz.ertel@outlook.com>; Author <author@noreply.github.com>
Temat: Re: [MysticRyuujin/guac-install] E: Unable to locate package libmysql-java tomcat7 (#104)


There's a reddit post that suggests we need to change to libmariadb-java for 19.10 but tomcat7 I'm not sure, can you run a: apt-cache show tomcat7

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMysticRyuujin%2Fguac-install%2Fissues%2F104%3Femail_source%3Dnotifications%26email_token%3DAD6OPA66ZT3L4VOO75HXGCDRCXDTJA5CNFSM4KUYW6K2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOELW37KI%23issuecomment-586006441&data=02%7C01%7C%7Cc9b9e4b474394f12b0c608d7b0d555e7%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637172302617848686&sdata=2BBYSEYJLwzWtsgv1dcVEnZGQ6R3d981tlFq4VnPlDA%3D&reserved=0>, or unsubscribe<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAD6OPA4RVILUEEBZUY5IEDLRCXDTJANCNFSM4KUYW6KQ&data=02%7C01%7C%7Cc9b9e4b474394f12b0c608d7b0d555e7%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637172302617858680&sdata=9hhnuhYVcG52EGuQoun5m%2BbJwH6bZwHpzCd7cbteOdU%3D&reserved=0>.
Od: Chase Wright<mailto:notifications@github.com>
Wysłano: czwartek, 13 lutego 2020 23:37
Do: MysticRyuujin/guac-install<mailto:guac-install@noreply.github.com>
DW: lukaszertel<mailto:lukasz.ertel@outlook.com>; Author<mailto:author@noreply.github.com>
Temat: Re: [MysticRyuujin/guac-install] E: Unable to locate package libmysql-java tomcat7 (#104)


There's a reddit post that suggests we need to change to libmariadb-java for 19.10 but tomcat7 I'm not sure, can you run a: apt-cache show tomcat7

**Comment by MysticRyuujin** [comment]
Well the 18.04 LTS is the recommended version

**Comment by lukaszertel** [comment]
Thanks, I will try.

**Comment by MysticRyuujin** [comment]
Fix in #105
